src/Pages/SignUp/SignUp.js

```diff
@@ -1,9 +1,255 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import CheckList from './CheckList';
+import './SignUp.scss';
 
 class SignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name_box: '',
+      number: '',
+      id: '',
+      pw: '',
+      confirm: '',
+      confirmNum:true,
+      pw_again:'',
+      pwAlert: true,
+      numberValid: true,
+      allChecked: false,
+      checked0: false,
+      checked1: true,
+      checked2: false,
+      checkList:[],
+    };
+  }
+
+  handleInputValue = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+    const {number} = this.state;
+    const checkNumber = !isNaN(number)&& number.length>=9 ;
+
+    
+    this.setState({
+    numberValid: checkNumber? false: true,
```

Comment:
삼항연산자를 이용해 false, true를 명시하지 않아도 checkNumber의 불리안 값이 바로 state에 적용되도록 할 수 있겠네요!
```suggestion
    numberValid: checkNumber;
```